### DISCUSSION TITLE
Where's the Trash?

### DISCUSSION THREAD

**Discussion by harabedian** [elementary/files#2430] [Q&A]
Maybe it's just me but I don't see a "Trash" icon anywhere in the "Files" app.  I only see my mounted drives.  If I go into the GNOME version of "Files" there is a "Trash" there, but not in the Elementary version of "Files".  See pic:

![Files Apps Elementary](https://github.com/elementary/files/assets/14143329/95b8affd-4c3c-4d8b-9b8c-b18f70e5f610)

**Comment by danirabbit** [comment]
Trash is in the Bookmarks section of the sidebar that you have collapsed. If you click on that header "Bookmarks" it will expand and show your special folders including Trash.

You can also type "Trash" in the path bar to navigate there if you prefer to keep your bookmarks collapsed
